about_solid.html

```diff
@@ -41,8 +41,8 @@ <h1>About Solid</h1>
                 <p class="mt-3">Solid is an open standard for structuring data, digital identities, and applications on
                     the Web.</p>
                 <p>Solid aims to support the creation of the Web as Sir Tim Berners-Lee originally envisioned it
-                    when he invented the Web at CERN 1989. Tim sometimes refers to Solid as “the third layer of
-                    the web” — or Web3.0 — because Solid integrates a new layer of standards into the Web we
```

Comment:
My change 1: I think "at CERN in 1989" is what was meant here.

My change 2: I don't think Tim ever used the term 'layer but I did find https://solidproject.org/take3 so used "the web - take 3" here.